app/editor/src/components/layout/styled/Layout.tsx

```diff
@@ -31,6 +31,15 @@ export const Layout = styled.div`
     justify-content: center;
   }
 
+  header {
```

Comment:
Could maybe use a switch statement just for easier readability

Reply:
Updated, thanks